tests/baselines/reference/declarationEmit_nameConflictsWithAlias.js

```diff
@@ -9,7 +9,7 @@ export module M {
 //// [declarationEmit_nameConflictsWithAlias.js]
 var M;
 (function (M) {
-    M.w; // Gets emitted as C.I, which is the wrong interface
+     // Gets emitted as C.I, which is the wrong interface

```

Comment:
This is pretty confusing now that the export is omitted, but I think we can fix it down the line.